tests/bot.test.js

```diff
@@ -0,0 +1,4 @@
+const { TestAdapter, ActivityTypes, TurnContext, ConversationState, MemoryStorage, UserState } = require('botbuilder');
```

Comment:
möchtest du auf dem Memory Storage bleiben? 
was gäbe es für alternativen? wieso wählst du was du wählst? -> in die Doku.